package.json

```diff
@@ -1,18 +1,64 @@
 {
   "name": "@cucumber/screenplay",
-  "version": "1.0.0",
+  "version": "1.0.0-next.1",
   "description": "Utilities to use screenplay pattern easily with Cucumber",
   "main": "dist/src/index.js",
   "types": "dist/src/index.d.ts",
   "dependencies": {},
+  "peerDependencies": {
+    "@cucumber/cucumber": "^7.0.0"
```

Comment:
Will people still be able to use another version of Cucumber-JS or will this force them to use at least 7.0 ?

Reply:
They will need at least 7.0.0. If we want to test with older versions we should set up a build matrix in Ci and I feel a bit lazy to do that....

Reply:
Ok works for me, I just wanted to be sure.